spec/iot-js-spec.md

```diff
@@ -98,11 +95,9 @@ dictionary OicDeviceSettings {
   USVString url;  // host:port
   OicDeviceInfo info;
   OicDeviceRole role;
-  OicConnectionMode connectionMode;  // mapping to “Quality of Service” in IoTivity
```

Comment:
It is implementation detail until we have a use case for it in clients. We can add it back any time. If you have a use case, could add it back now :).